lib/shopify-cli/core/monorail.rb

```diff
@@ -99,7 +99,7 @@ def build_payload(start_time, end_time, args, err = nil)
 
               if Project.has_current?
                 project = Project.current
-                payload[:api_key] = project.env.api_key
+                payload[:api_key] = File.exist?("#{Dir.pwd}/.env") ? project.env.api_key : ''
```

Comment:
Monorail ideally shouldn't have to know about the structure of the a project & its environment. I think it might be better to have a `project.has_env?` method, which could be useful later anyway since projects don't require an env. What do you think?

This is probably test worthy too since we have logic that alters the payload

Reply:
You are looking for `Project.has_current?`